cashu/core/db.py

```diff
@@ -127,7 +127,7 @@ def __init__(self, db_name: str, db_location: str):
                 print(f"Creating database directory: {self.db_location}")
```

Comment:
Please revert these changes (not relevant for this PR)

Reply:
Well, i agree but it is relevant as before i made those changes, the `api` was hanging trying to connect to the mint. I can exactly recall why, i'll try to go around it but i think this was the way.